packages/app-store/routing-forms/lib/getSerializableForm.ts

```diff
@@ -63,41 +144,39 @@ export async function getSerializableForm<TForm extends App_RoutingForms_Form>({
     description: f.description,
   }));
 
-  const finalFields = fields.map((field) => getFieldWithOptions(field));
-
-  let teamMembers: SerializableFormTeamMembers[] = [];
+  let teamMembers: NormalizeTeamMembersResult = [];
   if (form.teamId) {
-    teamMembers = await prisma.user.findMany({
-      where: {
-        teams: {
-          some: {
-            teamId: form.teamId,
-            accepted: true,
+    teamMembers = normalizeTeamMembers(
+      await prisma.attributeToUser.findMany({
```

Comment:
Doesn't show any member for me now in the dropdown, I think it's because of these changes: 
(I don't have any attributes set up)
![Screenshot 2024-10-23 at 4 44 32 PM](https://github.com/user-attachments/assets/c8a9678f-5ce0-41bc-bbe3-340144e7ae6c)